src/hooks/useUserPositions.ts

```
@@ -1,6 +1,8 @@
 import { useCallback } from 'react';
 import { useQuery, useQueryClient } from '@tanstack/react-query';
 import { Address } from 'viem';
+import { useConfig } from 'wagmi';
```

ESLint

[error] 'useConfig' is defined but never used.

(@typescript-eslint/no-unused-vars)